exporter/datadogexporter/README.md

```diff
@@ -2,11 +2,14 @@
 
 | Status                   |                  |
 | ------------------------ |------------------|
-| Stability                | [beta]           |
-| Supported pipeline types | traces, metrics  |
+| Stability                | traces [beta]    |
+|                          | metrics [beta]   |
+|                          | logs [alpha]     |
+| Supported pipeline types | traces, metrics, logs|
 | Distributions            | [contrib], [AWS] |
 
-This exporter sends metric and trace data to [Datadog](https://datadoghq.com). For environment specific setup instructions visit the [Datadog Documentation](https://docs.datadoghq.com/tracing/setup_overview/open_standards/otel_collector_datadog_exporter/).
+
+This exporter sends metric ,trace data and logs to [Datadog](https://datadoghq.com). For environment specific setup instructions visit the [Datadog Documentation](https://docs.datadoghq.com/tracing/setup_overview/open_standards/otel_collector_datadog_exporter/).
```

Comment:
`metric ,trace data and logs` -> `metrics, traces, and logs` or `metric, trace, and log data`